plugins/jobs/src/js/components/JobsBreadcrumbs.js

```diff
@@ -4,15 +4,15 @@ import PropTypes from "prop-types";
 import React from "react";
 import { Tooltip } from "reactjs-components";
 
+// everything that is references as #ALIAS/something has to be refactored once our DI system is in place
```

Comment:
What is the purpose of this comment? I get the message, but this information seems not relevant to me in this file.

Reply:
We want to change these once we are able to, the idea is that whoever stumbles upon these imports know immediately, that they are not best practice but legacy compatibility.

Reply:
I don't know if this really has the effect you aim for. I would rather remove them, otherwise you have these stray comments that appear when you plan for something but the plan changes. You see them a lot in older parts of our code base